plugins/modules/vm_snapshot_attach_disk.py

```diff
@@ -242,9 +262,35 @@ def main() -> None:
                 type="int",
                 required=True,
             ),
+
+            # Maybe the user will want to keep the destination VM shut down if, for example he has more tasks on
+            # this VM and wants to make it go quicker by keeping the VM shut down
+            #   - keep or remove this parameter?
+            #       - if keep, what could be a better name for it?
+            #           - dont_reboot_destination_vm
+            #           - keep_destination_vm_shutdown
+            #           - keep_destination_vm_off
+            #           - ignore_destination_vm_reboot --> False: reboot, True: no reboot
+            #           - ...?
+            reboot_destination_vm=dict(
+                type="bool",
+                default=True,
+            ),
+
+            # These two parameters must be present in order to use the VM functions from module_utils/vm
+            #   - see:
+            #       - vm_snapshot -> power_off_vm
+            #       - vm_snapshot -> power_up_vm
+            force_reboot=dict(
+                type="bool",
+                default=False,
+            ),
+            shutdown_timeout=dict(  # make this default to 300? (300 --> 5 minutes)
+                type="float",
+                default=30,
+            )
```

Comment:
Should the default be 300 seconds (as seen in the ``vm`` module) or keep it at 30?

Reply:
Also, if you search for string `scale_computing.hypercore.force_reboot` in plugins/modules, you will see we already have a documentation fragment for `force_reboot` param. Please try to reuse it.

Hopefully also code to do "nice and if needded and allowed also forced VM shutdown" can be reused.

Reply:
module_utils/vm.py, class VM, method do_shutdown_steps - this should be used. You can look at modules/vm.py, method _set_boot_order for example usage. Hope this helps.